lighthouse-viewer/app/src/lighthouse-report-viewer.js

```diff
@@ -397,6 +403,23 @@ class LighthouseViewerReport {
       this.closeExportDropdown();
     }
   }
+
+  /*
+   * Initializes of a `message` listener to respond to postMessage events.
+   */
+  listenForMessages() {
+    window.addEventListener('message', e => {
```

Comment:
I can see cases where we'd want the same page to update itself again. For example, when the DT opens the tab, it could re-`postMessage()` to the existing open tab.